plugins/module_utils/rds.py

```diff
@@ -181,20 +230,43 @@ def wait_for_cluster_status(client, module, db_cluster_id, waiter_name):
         module.fail_json_aws(e, msg="Failed with an unexpected error while waiting for the DB cluster {0}".format(db_cluster_id))
 
 
+def wait_for_snapshot_status(client, module, db_snapshot_id, waiter_name):
```

Comment:
Could you please create a new unit test file for this module_util and cover this new function?  Eventually we should try to cover the rest of the util but it's a lot easier to ask for folks to contribute tests for changes if the test file already exists.

We should also run the integration tests for the RDS modules with a depends-on for this change, to ensure there's no regressions.

Reply:
@jillr Done that!